src/network/networkd-sysctl.c

```diff
@@ -770,3 +789,14 @@ int config_parse_ip_forward_deprecated(
                    "and the same settings in .network files for per-interface setting.");
         return 0;
 }
+
+static const char* const ipv4_force_igmp_version_table[_IPV4_FORCE_IGMP_VERSION_MAX] = {
+        [IPV4_FORCE_IGMP_VERSION_NO]     = "no",
+        [IPV4_FORCE_IGMP_VERSION_1]      = "1",
+        [IPV4_FORCE_IGMP_VERSION_2]      = "2",
+        [IPV4_FORCE_IGMP_VERSION_3]      = "3",
```

Comment:
alignment on `=` is good. But there are too many spaces before it. there should be just one space between the `]` and the `=` on the line with the longest enum name

Reply:
Ok. I update the pull request